Source/Scene/PointCloud3DTileContent.js

```diff
@@ -781,8 +785,11 @@ define([
         var hasNormals = content._hasNormals;
         var hasBatchIds = content._hasBatchIds;
         var backFaceCulling = content._backFaceCulling;
+        var normalShading = content._normalShading;
         var vertexArray = content._drawCommand.vertexArray;
 
+        var normalsEnabled = hasNormals && normalShading;
```

Comment:
`backFaceCulling` and `normalShading` are independent of each other but both require normals. So `normalsEnabled` should be true if `hasNormals` is true and at least one of the options is true. When appending the back face culling code it will need to check `normalsEnabled && backFaceCulling` and when appending the normal shading code it will need to check `normalsEnabled && normalShading`.

The original idea I had was to keep most of the code as is and just wrap the normal shading code with `hasNormals && normalShading`.